packages/tasit-action/src/NFT.test.js

```diff
@@ -4,6 +4,11 @@ import NFT from "./NFT";
 // See https://github.com/tasitlabs/TasitSDK/issues/138
 const fullNFTAddress = "0x0E86f209729bf54763789CDBcA9E8b94f0FD5333";
 
+// Note: Under the current `tasit-contracts` setup SimpleStorageWithRemoved aways will deployed with this address
+// See https://github.com/tasitlabs/TasitSDK/issues/138
+const simpleStorageWithRemovedAddress =
```

Comment:
```suggestion

// Note: This contract is used not because of the particulars of this contract, but
// just because we needed a contract to send an NFT to and have it fail
// when using safeTransferFrom because this contract doesn't implement
// onERC721Received
const simpleStorageWithRemovedAddress =
```